getflywheel/local-addon-broken-link-checker
# chore (LOC-4487): Update Add-ons (Link Checker) to use latest version of @getflywheel/local

**Overview**

ACs:
- Each add-on references the latest Local API -> @getflywheel/local@6.6.0 -> DONE!
     - Will also mean a bump to the minimum version of Local
- Each add-on references the latest Local Components -> local-components@17.6.3 -> DONE!
- Visual changes are made in each add-on to be congruent with new Local Components version ->  Done!

**Visual appearance:**

To test:
1. Ensure that the Link Checker addon is not installed in Local (Plugins tab)
2. Clone this repo in your Local addons folder -> `cd "Library/Application Support/Local/addons"`
3. Navigate to the newly created `local-addon-broken-link-checker` directory and run `yarn install` and `yarn build`
4. Open Local (current release or local build) and go to the plugins tab, verify that the Link Checker is now installed (v1.1.2)
5. Then open up WP Admin for a site (any working site) and add a defective link in a new post (e.g. `http:localwp.com/garbage text`)
6. On the site page, in Local, navigate to 'Tools' and then 'Link Checker' -> Scan for links
7. Verify the functionality and visuals

<img width="1342" alt="image (1)" src="https://user-images.githubusercontent.com/59181296/204641014-bb395073-ed1d-49ac-b3e9-fec797028518.png">
